packages/fuselage/src/components/Callout/Callout.spec.tsx

```diff
@@ -1,13 +1,52 @@
 import { composeStories } from '@storybook/testing-react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import React from 'react';
 
+import { Callout } from './Callout';
 import * as stories from './Callout.stories';
 
-const { Default } = composeStories(stories);
+const { Default, WithDescriptionOnly, Info, Success, Warning, Danger } =
+  composeStories(stories);
 
 describe('[Callout Component]', () => {
-  it('renders without crashing', () => {
-    render(<Default />);
+  describe('Story renders without crashing', () => {
```

Comment:
I think just the `Default` variant is enough for this first test

Reply:
I did some refactoring